pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -25,6 +25,21 @@ export const LogTable: React.FC<LogTableProps> = ({ entityId, sourceId, endpoint
     handleSetLogs();
   }, [API, entityId, endpointId, sourceId]);
 
+  React.useEffect(() => {
+    handleSetDocumentation();
+  });
+
+  const handleSetDocumentation = (): void => {
+    API.Documentation.get("logtables")
+      .then((res) => {
+        setDocumentation(res.data.content);
```

Comment:
This `state` (`const [documentation, setDocumentation] ...`) does not exist and therefore the application is breaking.

Also, when re-implementing this state please check the modal, because it's not implementing `documentation` as content anymore either.

**Note:** this was intentionally removed because this functionality was commented out in `development` instead of removed.